src/components/TabMenu/TabMenu.js

```diff
@@ -11,17 +11,20 @@ const defaultTabs = [
 ];
 
 class TabMenu extends PureComponent {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
     this.state = {
-      selectedIndex: 0,
+      selectedIndex: props.selectedIndex || 0,
```

Comment:
`getDerivedStateFromProps` to set `selectedIndex`?

Reply:
We actually already have that [here](https://github.com/bpanel-org/bpanel-ui/pull/40/files#diff-9375fa9b13d4cae87d543ab1037836beR43).  This is to initialize it with props, but I'm not sure if that's necessary since getDerivedState should take care of it... I'll remove this change.